lib/hal/fileio.c

```diff
@@ -89,6 +89,12 @@ static char *getCurrentDirString()
  */
 int XConvertDOSFilenameToXBOX(const char *dosFilename, char *xboxFilename)
 {
+	// Allow Xbox filenames directly
+	if (!memcmp(dosFilename, "\\Device\\", 8)) {
```

Comment:
The documentation at the function header has to be adapted.

Reply:
Note: invalid paths may make it through this check (e.g. if I just pass in "\\Device\\"). Though better error checking should happen down the road so I think it's fine for now.